### PR TITLE
feat(emoji): added essential emojis

### PR DESCRIPTION
**Pull request by NickBurnett**
added all the cool ones

### CODE REVIEW

**Review by lacjnhi — APPROVED** [review]
lgtm!

File: scripts/constants.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ const submitFailStyle = 'background-color: #e32636; color: white; padding: 4px 8
 const difficulty_colors = ['green', 'rgb(239, 108, 0)', 'red'];
 
 const replacements = {
+  ":biggwiggle:": "https://cdn.discordapp.com/emojis/1022965811498979378.gif?size=96&quality=lossless",
+  ":bigwiggle:": "https://cdn.discordapp.com/emojis/1022965811498979378.gif?size=96&quality=lossless",
+  ":thonk:": "https://cdn.discordapp.com/emojis/539305279217598474.webp?size=96&quality=lossless",
+  ":cope:": "https://cdn.discordapp.com/emojis/752324497935171664.webp?size=96&quality=lossless",
```
Comment (sunnyguan) [inline scripts/constants.js]: image

**Review by sunnyguan — CHANGES_REQUESTED** [review]

File: scripts/constants.js (hunk 1 above)
Comment (NickBurnett) [reply scripts/constants.js]: Great point @sunnyguan! I'll make that change immediately. Sorry for the delay.

**Comment by sunnyguan** [comment]
Please remove the 1 star review :)

**Pull request merged**